test/e2e/push_test.go

```diff
@@ -116,15 +116,26 @@ var _ = Describe("Podman push", func() {
 		push := podmanTest.Podman([]string{"push", "-q", "--tls-verify=false", "--remove-signatures", ALPINE, "localhost:5000/my-alpine"})
 		push.WaitWithDefaultTimeout()
 		Expect(push).Should(Exit(0))
+		Expect(len(push.ErrorToString())).To(Equal(0))
```

Comment:
not worth the repush but please use `To(HaveLen(0))` or `To(Equal(""))` in the future, this creates much better error messages